### ISSUE TITLE
Not working/starting on macOS Mojave

### ISSUE THREAD

**Issue by ErdmannOle** [KAMIKAZEUA/NativeDisplayBrightness#7]
Hi,

Unfortunately the app is not working for me anymore. I used it without any problems on macOS Sierra, updated today to macOS Mojave 10.14.6 and the app won't even start. I tried reinstalling version 1.0 and 1.1 but I still get nothing.

I'm on a MacBook Pro 2012, Intel HD 4000. Display is a Dell U2414H, although that shouldn't be the problem as it worked just fine before.

**Comment by Anywhere-Music-Player** [comment]
Can you run project in Xcode and post a screenshot when it’s crash?

**Comment by ErdmannOle** [comment]
Ok, so I don't know if this will help you but here is what happened when I opened your project in Xcode. When running the code I got presented with some warnings that I was able to correct by following the menu prompts. After applying those changes the app worked perfectly again.

I documented all necessary steps:

![Bildschirmfoto 2019-08-03 um 17 35 26](https://user-images.githubusercontent.com/6563832/62414117-205dcf80-b618-11e9-8960-3dbbfac47157.png)
![Bildschirmfoto 2019-08-03 um 17 36 09](https://user-images.githubusercontent.com/6563832/62414118-205dcf80-b618-11e9-86f5-0827760a80d3.png)
![Bildschirmfoto 2019-08-03 um 17 36 38](https://user-images.githubusercontent.com/6563832/62414119-205dcf80-b618-11e9-9afb-2bf1bc3f7ba8.png)
![Bildschirmfoto 2019-08-03 um 17 36 47](https://user-images.githubusercontent.com/6563832/62414120-20f66600-b618-11e9-809b-f7f5ab2544d1.png)
![Bildschirmfoto 2019-08-03 um 17 37 55](https://user-images.githubusercontent.com/6563832/62414121-20f66600-b618-11e9-8e3d-a0f0f97d1fba.png)
![Bildschirmfoto 2019-08-03 um 17 38 23](https://user-images.githubusercontent.com/6563832/62414122-20f66600-b618-11e9-9ced-f9dbb7a63dec.png)
![Bildschirmfoto 2019-08-03 um 17 38 46](https://user-images.githubusercontent.com/6563832/62414123-20f66600-b618-11e9-933e-64e1af420582.png)

**Comment by Anywhere-Music-Player** [comment]
This is just basic warnings and migration, which is non related. 
Just open downloaded project from github and running it without any changes at all, what result?

**Comment by ErdmannOle** [comment]
With the unchanged code all it says now is _Build Succeeded_ and the app starts working. The warnings I talked about earlier are still there, so that really isn't the problem. Shortly after I get a prompt from macOS to enable the app in the Accessibility menu. No matter if I enable it or not, the app keeps working.

Still, if I want to open the actual app downloaded from the releases tab, it doesn't open. All it does is to animate the icon like it would open, but absolutely nothing happens after that.

Sorry that I can't describe or show it better. Maybe you can give me more detailed instructions? :/

**Comment by Anywhere-Music-Player** [comment]
This just looks like something else. if the application is already running unchanged this should not be related to it. Check your Login Items, there should be some more than 1 this app. Maybe this causes the issue. Also, ideally you should have only 1 app instance on your Mac as .app

**Comment by ErdmannOle** [comment]
I checked the login items and there were several entries for NativeDisplayBrightness indeed. Although getting rid of those didn't seem to solve the problem.

What I believe I did now is to have replaced the .app with the debug file (?) from Xcode. I think this increased the file size slightly but I'm not sure about that. This is what the info tab for the app looks like now:
![Bildschirmfoto 2019-08-07 um 15 55 02](https://user-images.githubusercontent.com/6563832/62628745-32978080-b92c-11e9-91eb-9877885c4b2c.png)
Anyway, all seems to work now. Thank you very much for your assistance.

BTW: If you'd like a German translation for your app I'd be glad to help. For free of course an no need to credit whatsoever. Let me know if you are interested. Thanks again!
